halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/azure/AzureEditAccountCommand.java

```diff
@@ -94,6 +105,12 @@ protected Account editAccount(AzureAccount account) {
     account.setDefaultKeyVault(isSet(defaultKeyVault) ? defaultKeyVault : account.getDefaultKeyVault());
     account.setPackerResourceGroup(isSet(packerResourceGroup) ? packerResourceGroup : account.getPackerResourceGroup());
     account.setPackerStorageAccount(isSet(packerStorageAccount) ? packerStorageAccount : account.getPackerStorageAccount());
+
+    try {
+      account.setRegions(regions);
+    } catch (IllegalArgumentException e) {
+      throw new IllegalArgumentException("Set either --regions or --[add/remove]-region");
```

Comment:
Doesn't look like `--add-region` or `--remove-region` is supported, but they are referenced in the error message.

Reply:
fixed